make/go/base.mk

```diff
@@ -178,6 +178,9 @@ postgenerate::
 .PHONY: licensegenerate
 licensegenerate::
 
+.PHONY: format
```

Comment:
I haven't followed upstream changes, but I assume there were some changes..

Reply:
Yep - these were upstream changes to makego to optimize CI build time and also support Knit.